package functional.tests.core.Exceptions;

/**
 * Created by Dimitar on 10/26/2015.
 */
public class AppiumException extends Exception {
    public AppiumException(String message) {
        super(message);
    }
}